src/com/mercado/model/user/User.java

```diff
@@ -0,0 +1,15 @@
+package com.mercado.model.user;
+
+import com.mercado.model.BaseEntity;
+
+public abstract class User extends BaseEntity {
```

Comment:
Why would you have `User` as an abstract class?

Reply:
becouse a don't want use this directly. I just will use the childs like chef,admin, cahier , ...

Reply:
Humm I think `User` is a good candidate to be instantiable. `User` is a really wide and generic word.

My suggestion would be to set it as `non-abstracr class`, but I would let you decide that since in the future, I'm pretty sure you will change it since you may have normal users login into your application.